packages/cli-kit/src/public/node/api/webhooks.ts

```diff
@@ -0,0 +1,43 @@
+import {graphqlRequestDoc} from './graphql.js'
+import {appManagementFqdn} from '../context/fqdn.js'
+import Bottleneck from 'bottleneck'
+import {Variables} from 'graphql-request'
+import {TypedDocumentNode} from '@graphql-typed-document-node/core'
+
+// API Rate limiter
+// Jobs are launched every 150ms
+// Only 10 requests can be executed concurrently.
+const limiter = new Bottleneck({
+  minTime: 150,
+  maxConcurrent: 10,
+})
+
+/**
+ * Executes an org-scoped GraphQL query against the App Management API.
+ * Uses typed documents.
+ *
+ * @param query - GraphQL query to execute.
+ * @param token - Partners token.
+ * @param variables - GraphQL variables to pass to the query.
+ * @returns The response of the query of generic type <T>.
+ */
+export async function webhooksRequest<TResult, TVariables extends Variables>(
+  query: TypedDocumentNode<TResult, TVariables>,
+  token: string,
+  variables?: TVariables,
+): Promise<TResult> {
+  const api = 'Webhooks'
+  const fqdn = await appManagementFqdn()
```

Comment:
Kind of a nit but I wonder if we should rename `appManagementFqdn` since this FQDN is now shared amongst several remote services (app management, core webhooks, functions). We could still expose `appManagementFqdn` but it could directly delegate to this new name. It would avoid any ambiguity that the webhooks commands are hosted on the app management API as they are not.

I'm not sure what the best name would be though. `nonShopScopedShopifyFqdn`? I don't really like that either haha.

Reply:
there's `partnersFqdn` => partners.shopify.com and `developerDashboardFqdn` => dev.shopify.com. How about `appFqdn` or `appApiFqdn`?